Week-1/cattyboard/cattyboard.htm

```diff
@@ -0,0 +1,187 @@
+<html>
+<style>
+  * {
+    box-sizing: border-box;
+  }
+
+  header {
+    float: left;
+    display: flex;
+    background-color: #3a3b3c;
+    padding-left: 25px;
+    width: 100%;
+    text-align: left;
+
+    color: white;
+  }
+
+  nav {
+    position: absolute;
+    float: left;
+    width: 3em;
+    height:85%;
+    background: #a9a9a9;
+    padding: 10px 10px;
+  }
+  li button {
+
+    border: None;
+    outline: None;
+    background: none;
+
+  }
+  p button {
+    padding-left:2em;
+    border: None;
+    outline: None;
+    background: none;
+
+  }
+  nav ul {
+    list-style-type: none;
+    padding: 0;
+  }
+
+  article {
+    float:center;
+    padding: 80px 10px;
+    width: 80%;
+    background-color: #fff;
+    height: 200px;
+    text-align: center;
+  }
+  table {
+           border-collapse: collapse;
+           width: 100%;
+       }
+  td {
+    width:90%
+    float:center;
+    align: center;
+    padding: 10px 10px;
+  }
+  tr {
+    padding-left: 40px;
+  }
+  tr:nth-child(even) {
+            background-color: #a9a9a9;
+        }
+  section:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  footer {
+    position: absolute;
+    bottom: 0px;
+    height:15%;
+    width: 100%;
+    background-color: #3971ac;
+    vertical-align: bottom;
+    padding: 20px;
+    text-align: center;
+    color: white;
+  }
+  li {
+
+    padding-top:4em;
+  }
+  .chartimg {
+    float: left;
+    padding: 2em;
+    width: 15%;
+    height: 280px;
+  }
+  .fchartimg {
+    float: left;
+    padding-top:15px;
+    width: 15%;
+    display: flex;
+    display: table-cell;
+    vertical-align: top;
+  }
+  .index{
+    padding-left: 8%
+  }
+  .charttext {
+    float: left;
+    padding: 120px;
+    width: 80%;
+    height: 80px;
+    display: flex;
+    display: table-cell;
+    vertical-align: top;
+  }
+</style>
```

Comment:
Why is style outside of head ?

Reply:
It still seems to work, however now moved inside head tags.